package.json

```diff
@@ -30,9 +32,12 @@
     "vinyl": "^0.4.6",
     "webpack": "^1.6.0"
   },
-  "devDependencies": {},
+  "devDependencies": {
+    "gulp-jscs": "^1.4.0",
+    "gulp-jshint": "^1.9.2"
+  },
   "scripts": {
-    "test": "gulp smoketest && mocha test/*.test.js && npm run browsertest",
+    "test": "gulp smoketest && mocha test/*.test.js && npm run browsertest && gulp lint-test",
```

Comment:
Do you think you could remove this for now, solely so that our tests aren't broken while you're fixing the errors raised by `lint-test`? Then once you've fixed them, we can re-add it.